client-app/core/composables/useFieldValidationSchema.ts

```diff
@@ -0,0 +1,30 @@
+import { useI18n } from "vue-i18n";
+import type { NumberSchema } from "yup";
+
+function mutateQuantityFieldSchema(schema: NumberSchema, minQuantity?: number, maxQuantity?: number): NumberSchema {
```

Comment:
1. File name and composable name should match
2. Move it to cart comporables
3. Currently it's utility and not composable, because [composable should be stateful](https://vuejs.org/guide/reusability/composables.html#what-is-a-composable), but we probably want to move quantity validation schema here fully, because quantity type (number), label, error message and so on are equal in all places.